components/ContactForm/ContactForm.js

```diff
@@ -100,75 +86,50 @@ export function ContactForm() {
-              name="name"
-              id="name"
-              value={form.name}
-              onChange={handleChange}
-              onBlur={handleOnBlur}
-            />
-            {errors.name && <span>{input_text_message}</span>}
-          </label>
-        </p>
-        <p>
-          <label htmlFor="phone">
-            Your Phone number:
-            <input
-              type="text"
-              name="phone"
-              id="phone"
-              value={form.phone}
-              onChange={handleChange}
-              onBlur={handleOnBlur}
-            />
-            {errors.phone && <span>{input_phone_message}</span>}
-          </label>
-        </p>
-        <p>
-          <label htmlFor="email">
-            Your Email:
-            <input
-              type="email"
-              name="email"
-              id="email"
-              value={form.email}
-              onChange={handleChange}
-              onBlur={handleOnBlur}
-            />
-            {errors.email && <span>{input_email_message}</span>}
-          </label>
-        </p>
-        <p>
-          <label htmlFor="message">
-            Message:
-            <textarea
-              name="message"
-              id="message"
-              value={form.message}
-              onChange={handleChange}
-              onBlur={handleOnBlur}
-            />
-            <span>{`${form.message.length}/200`}</span>
-            {errors.message && <span>{input_textarea_message}</span>}
-          </label>
-        </p>
-        <ReCAPTCHA
-          sitekey={process.env.SITE_RECAPTCHA_KEY}
-          onChange={onChange}
+    <form
+      className={styles.contact_form}
+      name="contact"
+      onSubmit={handleSubmit}
+      noValidate
+    >
+      <div
+        className={classNames("mb-3", { [styles.is_error]: errors["email"] })}
+      >
+        <input
+          type="email"
+          name="email"
+          id="email"
+          value={form.email}
+          onChange={handleChange}
+          placeholder="e-mail"
+        />
+      </div>
+      <div
+        className={classNames("mb-5 position-relative", {
+          [styles.is_error]: errors["message"],
+        })}
+      >
+        <textarea
+          name="message"
+          id="message"
+          value={form.message}
+          onChange={handleChange}
+          rows="5"
+          placeholder="message"
         />
-        <p>
-          <button type="submit">
-            {translations[locale].contactForm.sendButton}
-          </button>
-        </p>
-      </form>
-    </>
+        <small id="emailHelp" className="form-text text-muted float-right">
+          {`${form.message.length}/200`}
+        </small>
+      </div>
+      <div className="text-right">
+        <button
+          className={styles.button}
+          disabled={isValidationFailed()}
+          type="submit"
+        >
+          {translations[locale].contactForm.sendButton}
+        </button>
+      </div>
+    </form>
```

Comment:
no i ja bym dal error mesage bo wpisujesz a nie wiesz dlaczego jest zle

Reply:
no ale przy tak prostych danych po co ? Email to wiadomo ze musi byc email a przy textarea masz wiadomosc 0/200

Reply:
Co najwyzej jako dodatek moze warto zrobic tekst pod textarea na czerwono